cmd/ledgerutil/main.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 const (
-	resultFilename = "result.json"
+	compareErrorMessage = "Ledger Compare Error: "
```

Comment:
If the snapshots are the same, it results in a message like:

"Ledger Compare Error: both snapshots public state hashes are same. Aborting compare"

As an end user, I wouldn't consider a consistent snapshot comparison to be a "Ledger Compare Error".  We should only indicate that it is an error if it is an unexpected scenario. Happy endings should result in a message rather than an error message.